web/mass_class/rtmp_bridge.py

```
@@ -0,0 +1,178 @@
+            )
+
+            # Notify teacher that streaming has started
+            await self.channel_layer.send(
+                teacher_channel,
+                {"type": "stream_started", "youtube_url": f"https://www.youtube.com/watch?v={youtube_key}"},
+            )
+
+            # Monitor process in background
+            asyncio.create_task(self._monitor_process(session_id, teacher_channel))
+
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)